ros2cli/test/test_daemon.py

```diff
@@ -110,6 +110,33 @@ def daemon_node():
     assert spawn_daemon(args=[], wait_until_spawned=5.0)
     with DaemonNode(args=[]) as node:
         yield node
```

Comment:
Is it sufficient to wait for discovery of the test node at the fixture level, here? E.g. wait until `get_node_names_and_namespaces()` discovers the test node.

Reply:
See https://github.com/ros2/ros2cli/pull/504/commits/559de8eba42069fe46e3e40bf6af9c5e9b68d1b8.